tools/SetupFlow/DevHome.SetupFlow.UnitTest/TestingScenarios/QuickstartPlayground.md

```diff
@@ -0,0 +1,36 @@
+# Quickstart Playground Tests
+If your code affects Quickstart Playground, please manually verify these scenarios.
+
+## Scenarios
+Please make sure to verify all of these scenarios. These apply to both Windows 10 and Windows 11.
+
+### First Run and First Setup Experience (Machine Has Never Run Quickstart Playground Before and Needs Dependencies Configured)
+1. Launch Dev Home
+1. Navigate to 'Settings -> Experimental features', and validate that Quickstart Playground is off by default. Then turn it on.
+    1. Validate that the Dev Home Azure extension is brought down as part of turning on the feature. When the extension is ultimately installed, you should see a toast notification from the Store.
+1. Navigate to the machine configuration page and you should see a button for the Quickstart Playground feature. Click on that entry. 
+1. Select the Dev Container with OpenAI extension from the ComboBox.
+    1. You should see a ContentDialog with an Adaptive Card guiding you to install any missing dependencies (i.e., Visual Studio Code, Docker Desktop, Remote Containers extension)
+    1. Use the download links to set up any  missing dependencies, accepting all defaults.
+    1. If Docker Desktop was installed, you'll need to reboot the system, manually re-launch Dev Home, and navigate back to the Quickstart Playground page. 
+1. Once all of the application dependencies have been set up, you will see another Adaptive Card for entering in the OpenAI API key. Enter your key into the PasswordTextBox shown to you by the card.
```

Comment:
Just wondering would there ever be a need to reboot DevHome after entering the API key?

Reply:
Not to my knowledge. The API key is an implementation detail of the extension, and Dev Home technically doesn't know anything about it.